android/src/main/java/com/robinpowered/react/ScreenBrightness/ScreenBrightnessModule.java

```diff
@@ -61,13 +61,16 @@ public String getName() {
         return MODULE_NAME;
     }
 
-    @Override
-    public void onActivityResult(int requestCode, int resultCode, Intent intent) {
+    public void onActivityResult(Activity activity, int requestCode, int resultCode, Intent intent) {
         if (requestCode == writeSettingsRequestCode) {
             onPermissionResult();
         }
     }
 
+    public void onNewIntent(Intent intent) {
```

Comment:
What was this needed for?

Reply:
https://github.com/facebook/react-native/blob/9ee815f6b52e0c2417c04e5a05e1e31df26daed2/ReactAndroid/src/main/java/com/facebook/react/bridge/ActivityEventListener.java

Reply:
We're implementing `ActivityEventListener` here.